consensus/istanbul/backend/engine.go

```diff
@@ -95,15 +96,68 @@ var (
 	nonceAuthVote = hexutil.MustDecode("0xffffffffffffffff") // Magic nonce number to vote on adding a new validator
 	nonceDropVote = hexutil.MustDecode("0x0000000000000000") // Magic nonce number to vote on removing a validator.
 
-	inmemoryAddresses  = 20 // Number of recent addresses from ecrecover
-	recentAddresses, _ = lru.NewARC(inmemoryAddresses)
+	inmemoryBlocks             = 100 // Number of blocks to precompute validators' addresses
```

Comment:
This value is enough? 
Could you check the all block header can be preprocessed in normal cases?

```suggestion
	inmemoryBlocks             = 100 // Number of blocks to precompute validators' addresses
```

Reply:
I will increase size to 2048 suggested in https://github.com/klaytn/klaytn/pull/957/files#r629000344